upgrade/rendering-changes/4-0-0.md

```diff
@@ -0,0 +1,233 @@
+| `<tfoot>` | `<tfoot class="k-table-tfoot">` |
+| `<th>`  | `<th class="k-table-th">`  |
+| `<tr>` |  `<tr class="k-table-row">` |
+| `<td>` | `<td class="k-table-td">` |
+| `<tr class="k-grouping-row">` | `<tr class="k-table-group-row">` |
+
+### Grid Grouping
+
+* Removed `<div class="k-indicator-container">` in favor of `<div class="k-chip>`. The Group value is now rendered in a `<span class="k-chip-content">`. The close icon is now rendered in a `<span class="k-chip-actions">` container.
+* Changed the close icon from `Icon="x"` to `Icon="x-circle"`.
+* Removed the `k-grouping-header-flex` class from the grouping header.
+* Renamed the class of the empty `<div class="k-indicator-container">` to `<div class="k-grouping-drop-container">`.
+
+## ListView
+
+* Added `<div class="k-listview-item">` wrapper to Item Template.
+
+## MultiSelect
+
+* Added `k-text-ellipsis` class to the `<span class="k-chip-label>` element.
+* Removed `k-input-values` and `k-selection-multiple` classes from the `<div class="k-chip-list">` element.
+* Wrapped the `<div class="k-chip-list">` element in a `<div class="k-input-values">` container.
+
+## Notification
+
+* Removed `<div class="k-notification-wrap">` element.
+* Replaced closable Notification icon `<a class="k-icon k-i-x">` with a `<span class="k-icon k-i-x">`.
+* Wrapped the closable icon in a `<span class="k-notification-actions">` container.
+
+## Pager
+
+* Replaced `<ul class="k-pager-numbers">` with a `<div class="k-pager-numbers">`.
+* Replaced navigation buttons `<a class="k-link k-page-nav">` with flat buttons `<button class="k-pager-nav k-button k-button-{size} k-button-flat k-button-flat-base k-icon-button">`.
+* Replaced page numbers `<span class="k-link>` with flat primary buttons `<button class="k-button k-button-{size} k-button-flat k-button-flat-primary">`
+* Removed `k-floatwrap`, `k-link`, `k-pager-wrap` and `k-label` classes.
+
+## ProgressBar
+
+* Added `k-progressbar-progress` class to the `<div class="k-selected>` element.
+* Changed the inline style of the `<div class="k-progressbar>` from `--kendo-progressbar-progress` to `--kendo-progressbar-value`. (This style conrols the progress width/height.)
+
+## Splitter
+
+* Removed the `k-collapse-prev` and `k-collapse-next` classes from the `<span class="k-icon>`.
+
+## Toolbar
+
+* Added `k-toolbar-resizable` class to the main wrapping element of the component (`<div class="k-toolbar>`). Applicable when the Toolbar is responsive.
+* Renamed `k-overflow-anchor` class to `k-toolbar-overflow-button` class.
+* Added Toolbar-specific classes to all built-in tools(`k-toolbar-button`, `k-toolbar-toggle-button`, `k-toolbar-button-group` and more)
+* All template items are wrapped in `<div class="k-toolbar-item">`.
+* When the Toolbar is part of the Filter component, `FilterExpression` items have a `k-toolbar-item` class. Applicable only if said item is a complex one(containing either a select/input component).
```

Comment:
Do you think it might be clearer if we move this to a separate `Filter` section? This is strictly related to the Filter component and the fact it internally reuses the ToolBar(shares the markup) is an implementation detail.